.github/workflows/scripts-lint.yml

```diff
@@ -38,3 +38,24 @@ jobs:
         yarn install
         yarn lint
         yarn js-check
+  run-py-linters:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Install Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.12'
+
+    - name: Install Poetry
+      uses: snok/install-poetry@v1
+      with:
+        virtualenvs-create: true
+
+    - name: Install dependencies
+      working-directory: scripts/mirror-actors
+      run: poetry install --no-interaction --no-root
+
+    - name: Lint Python Code
+      working-directory: scripts/mirror-actors
+      run: poetry run pylint mirror_actors/ -f actions
```

Comment:
Could also add black as a formatter, and pylance as a type checker if you really want "production grade python". 

Might not be relevant for quick scripts

Happy for this to be deferred as an issue

Reply:
see [issue](https://github.com/ChainSafe/forest-iac/issues/390)